scripts/debian/build.sh

```diff
@@ -5,6 +5,18 @@
 set -eox pipefail
 
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+BUILD_DIR=${BUILD_DIR:-"${SCRIPTPATH}/../../_build"}
```

Comment:
This script was primarly used only in buildkite context.  In caller script we used combination of cd operations and SCRIPTPATH env vars to navigate to correct PWD before running this script. Now we don't want to meddle with cd as it might affect local dev PWD (which can be frustrating) . Therefore we left PWD intact and introduced BUILD_DIR env var which controls context

Reply:
Could we define it once and for all in `export-git-env-vars.sh`? The script `export-git-env-vars.sh` could be the file defining all exported variables for the whole monorepo.